prow/plugins/trigger/ic.go

```diff
@@ -120,13 +118,16 @@ func handleIC(c client, trigger *plugins.Trigger, ic github.IssueCommentEvent) e
 		}
 	}
 
-	if okToTest && ic.Issue.HasLabel(NeedsOkToTest) {
-		if err := c.GitHubClient.RemoveLabel(ic.Repo.Owner.Login, ic.Repo.Name, ic.Issue.Number, NeedsOkToTest); err != nil {
-			c.Logger.WithError(err).Errorf("Failed at removing %s label", NeedsOkToTest)
+	okToTestAdded := false
+	if isOkToTest {
+		if err := c.GitHubClient.AddLabel(ic.Repo.Owner.Login, ic.Repo.Name, ic.Issue.Number, okToTest); err != nil {
+			return err
 		}
-		err = clearStaleComments(c.GitHubClient, *pr, comments)
```

Comment:
Ok, so the idea here was to avoid listing comments completely (once some time has passed, to avoid a breaking update) and therefore I have altered the welcome comment to remain meaningful after `/ok-to-test` has been done.
But now I realize there is another comment, the one that is added after an untrusted PR received a `/test` from a non member, that should be changed as well if I want to avoid listing/cleaning comments... is that what you mean?

Reply:
No, I didn't realize that the message was supposed to remain appropriate even after `/ok-to-test` has been issued. Typically it is better to clean up these kinds of comments when the process has been satisfied to make the PR's comments easier to read, but `trigger` is a heavily used plugin and this will save API tokens in the common case so it seems worth it. Lets leave this as is.